src/easyreflectometry/project.py

```diff
@@ -132,20 +146,21 @@ def default_model(self):
         self._replace_collection(MaterialCollection(), self._materials)
 
         layers = [
-            Layer(material=self._materials[0], thickness=0.0, roughness=0.0, name='Vacuum Layer'),
-            Layer(material=self._materials[1], thickness=100.0, roughness=3.0, name='Multi-layer'),
-            Layer(material=self._materials[2], thickness=0.0, roughness=1.2, name='Si Layer'),
+            Layer(material=self._materials[0], thickness=0.0, roughness=0.0, name='Vacuum Layer', interface=self._calculator),
```

Comment:
Maybe you can make `CalculatorFactory` a singleton, so you don't carry this with every call, to Layer, Multilayer, Sample etc.?
Do you need more than one instance of this factory?

Reply:
You are most certainly right.  I have not yet fully understood how when we need to pass the factory and I decided to be very explicit to be sure that it hits all objects.

I would like to make the suggested change, but that will have to wait to another PR